### ISSUE TITLE
[Tableau Feedback] Create a properties builder script

### ISSUE THREAD

**Issue by jainam1995** [ActianCorp/actian_tableau_connector#6]
Since your connector was started, we've changed our samples to better reflect best practices for JDBC. You can read about the connection properties script in our [documentation](https://tableau.github.io/connector-plugin-sdk/docs/api-reference#connection-properties), and we have a [sample ](https://github.com/tableau/connector-plugin-sdk/tree/master/samples/plugins/postgres_jdbc)jdbc plugin that uses the [connection properties script](https://github.com/tableau/connector-plugin-sdk/tree/master/samples/plugins/postgres_jdbc). Note that you'll need to change your connection builder script to match the sample as well.

Basically, the formatted params are set int he connection properties script. This has implications in Tableau, and will make sure we treat the properties correctly (for example, not logging the password value).

Note that connection properties has two different possible return values, either a javascript object or a string formatted to have key=value. The javascript object is preferred since it does not have trouble with an equals sign in a property, but was only introduced in 2019.4.1.

Let me know you if have any questions, or if there are clarifications I can make.

**Comment by clach04** [comment]
@jainam1995 Can you confirm (or deny ;)) this https://github.com/tableau/connector-plugin-sdk/blob/master/samples/plugins/postgres_jdbc/connectionProperties.js is only for JDBC?

We're using connectionHelper in ODBC as well as JDBC:

  * https://github.com/ActianCorp/actian_tableau_connector/blob/master/actian_jdbc/connectionBuilder.js
  * https://github.com/ActianCorp/actian_tableau_connector/blob/master/actian_odbc/connectionBuilder.js

**Comment by clach04** [comment]
Internal ref II-5538

**Comment by hab6** [comment]
@jainam1995 I am following up on this old issue and specifically the [comment](https://github.com/ActianCorp/actian_tableau_connector/issues/6#issuecomment-597908708) by @clach04. Could you please confirm whether the **connection-properties** applies only to JDBC connectors or also to ODBC connectors? It is not clear to me since both JDBC and ODBC connectors use JavaScript `connectionBuilder.js` files.

**Comment by hab6** [comment]
Added tag `connection-properties` and file `connectionProperties.js` in the JDBC connector code via PR #26.
